patterns-modules/design-patterns-behavioral/src/test/resources/logback.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please rename to logback-test.xml

Reply:
Renamed to logback-test.xml. I will update the PR at the end of the process. Thanks!